ui/DesignSystem/ThreeDotsMenu.svelte

```diff
@@ -6,53 +6,37 @@
  LICENSE file.
 -->
 <script lang="ts">
-  import { fade } from "svelte/transition";
-  import type { SvelteComponent } from "svelte";
-
-  import Icon from "./Icon";
-  import Overlay from "./Overlay.svelte";
-  import Tooltip from "./Tooltip.svelte";
-  import Copyable from "./Copyable.svelte";
-
   interface MenuItem {
     title: string;
-    icon: typeof SvelteComponent;
+    icon: typeof Icon;
     event: () => void;
     tooltip?: string;
     dataCy?: string;
     disabled?: boolean;
   }
-  export let menuItems: MenuItem[];
-  export let headerTitle: string | undefined = undefined;
 
+  import { fade } from "svelte/transition";
```

Comment:
The imports should precede the `interface` definition.

Reply:
73727843fdc9ef0a6dd257ab0ae5b4eb865f95f9